### ISSUE TITLE
Feed count and facebook feed

### ISSUE THREAD

**Issue by dzulhelmi** [pavelk2/social-feed#148]
Hi,

How can i limit or add the feed count? Example I want to show 25 latest feeds on the page.
One more thing, should i change any configuration to make sure feeds from Facebook also appear? Currently it just show Instagram and Twitter.

**Comment by yp28** [comment]
If you're trying to load Facebook posts based on a hashtag, this is not supported in the Facebook API anymore since around April this year.

**Comment by dzulhelmi** [comment]
I dont mind by hashtag or not, as long as it displays from Facebook. How to display facebook posts?

**Comment by pavelk2** [comment]
@dzulhelmi did you try to specify 

```
accounts: "@teslamotors"
```

or you can also specify 
accounts: "!123456"
where 123456 is facebook page ID

**Comment by dzulhelmi** [comment]
Hi @pavelk2 ,
If i use that way, then I cannot use the query tags?

**Comment by yp28** [comment]
That's correct. As I previously mentioned, hashtags are no longer supported by the Facebook API.
